APACHE_PROPOSAL.md

```diff
@@ -0,0 +1,307 @@
+
+#### Community
+
+Baidu has been building a community around users and developers for the last three years, now we make use of GitHub as code hosting and community communication. There are 10+ contributors from non-Baidu companies like 360, Kedaxunfei, Huyazhibo. We hope to grow the base of contributors by inviting all those who offer contributions through The Apache Way. 
+
+#### Core Developers
+
+The core developers are already very experienced open source developers, they have managed the hugegraph community for three years, and they are contributors of Linux kernel, OpenStack, Ceph, RocksDB, Apache TinkerPop, Apache Hadoop and Apache Groovy.
+
+#### Alignment
+
+HugeGraph is compliance to [Apache TinkerPop 3](https://tinkerpop.apache.org/) framework, various complicated graph queries can be accomplished through [Gremlin](https://tinkerpop.apache.org/gremlin.html), a powerful graph traversal language. The TinkerPop ecosystem based on Java is relatively mature in terms of graph databases, we use Java language which is develop efficient and stable to develop upper-level components like graph engine, graph computing, graph api and graph tools; and we manage storage through JNI which is able to freely manage memory and execute efficiently.
+
+### Known Risks
+
+#### Project Name
+
+We have checked that the name is [suitable](https://github.com/hugegraph/hugegraph/issues/1646) and the project has legal permission to continuing using its current name. There is no one else found using this name through Google search.
+
+#### Orphaned products
+
+Due to the relatively small number of contributors, the project has a relatively small risk of becoming an orphan. However, the contributors have been operating in the spirit of open source for 3 years, and continue to develop new contributors to participate.
+
+#### Inexperience with Open Source:
+
+HugeGraph has been open sourced on GitHub for 3 years, during which contributors submitted code and documents in full compliance with open source specifications and requirements
+
+#### Length of Incubation:
+
+Expect to enter incubation in 6 months and graduate in about 2 years
+
+#### Homogenous Developers:
+
+The current list of committers includes developers from several different companies plus many independent volunteers. The committers are geographically concentrated in China now. They are experienced with working in a distributed environment in other open source project, e.g. OpenStack.
+
+#### Reliance on Salaried Developers:
+
+Most of the developers are paid by their employer to contribute to this project, but given some volunteer developers and the committers' sense of ownership for the code, the project would continue without issue if no salaried developers contributed to the project.
+
+#### Relationship with Titan/Janus Graph
```

Comment:
keep the section in "Known Risks" and add a new section to describe "Relationships with Other Apache Products:"